data/json/itemgroups/guns.json

```diff
@@ -200,11 +201,12 @@
       [ "ak47", 100 ],
       [ "ak74", 60 ],
       [ "an94", 40 ],
+      [ "arx160", 40 ],
+      [ "cane_rifle", 80 ],
```

Comment:
Since the cane rifle is new production, it wouldn't make sense to limit it to pawn shops.
If gun laws were liberal, cane rifles would be produced either as a novelty item, which I thought would fit rifle_obscure, or as a overgunned self-defense weapon used by people with more money than sense.
The reason I continue to ask about gun laws is the fact that new production weapons like the cane rifle and custom-made weapons like the auto TEC-9 (which was only a semiautomatic weapon irl, unless a gunsmith modded it) would only be common if 
1. gun smuggling is common in the future
2. gun laws relating to the ownership of several types of weapons were amended or repealed.